engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -124,7 +153,727 @@ public static StramElement fromValue(String value) {
     }
 
   }
-  
+
+  /**
+   * This is an enum which represents a type of configuration.
+   */
+  protected enum ConfElement
```

Comment:
All these @SuppressWarnings make this very hard to read // understand. Is there a way to clean this up to make it more human readable? I know @SuppressWarnings flags for methods may be defined above the method name, not sure if the same works here.